zarf.yaml

```diff
@@ -2,9 +2,9 @@ kind: ZarfInitConfig
 metadata:
   name: Zarf Official Init Package
   description: "Used to establish a new Zarf cluster"
+  architecture: amd64
```

Comment:
Does this mean we need to write a 2nd Zarf init config for ARM?

Reply:
Technically no--but if we want to make all the init package images (#214 #215), we would need 2 separate ones.  You can pass the `-a` flag to zarf to override the architecture, that came with 0.15, but this just makes the default more obvious since our injector stuff right now is all amd64 compiled only rn.